releng/docker/Dockerfile

```diff
@@ -7,34 +9,100 @@
 # repository you want on /zbm inside the container.
 
 # Use the official Void Linux container
-FROM ghcr.io/void-linux/void-glibc-full:latest
+FROM ghcr.io/void-linux/void-glibc-full 
+LABEL org.opencontainers.image.authors="ZFSBootMenu Team, https://zfsbootmenu.org"
+
+# 	   ZFSBootMenu commit hash, tag or branch name used by
+#	   default to build ZFSBootMenu images (default: master)
 ARG ZBM_COMMIT_HASH
+#      Include the specified Void Linux package in the image
+#      
+#      (multiple entries must be seperated by spaces)
+ARG PACKAGES=
+ARG XBPS_REPOS="https://repo-fastly.voidlinux.org/current"
+# 	 Include the specified Void Linux kernel series in the image; kernel
+#    series take the form <major>.<minor> and correspond to Void packages
+#    linux<kver> and linux<kver>-headers
+#
+#    Default: install 5.10, 5.15 and 6.1
+#	
+#    (multiple entries must be seperated by spaces)
+ARG KERNELS="linux5.10 linux5.15 linux6.1" 
+
+RUN mkdir -p /etc/xbps.d
+RUN <<EOF
+for repo in "${XBPS_REPOS}"; do
+  echo "repository=$repo" >> /etc/xbps.d/00-custom-repos.conf
+done
+EOF
 
 # Ensure everything is up-to-date
-RUN xbps-install -Suy xbps && xbps-install -uy
+RUN xbps-install -Suy xbps && xbps-install -uy bash
+SHELL ["/bin/bash", "-c"]
+
+# Install custom packages
+RUN <<-EOF
```

Comment:
`-` in heredoc syntax should only be necessary if the base indent is at least 1 tab, so i'd either remote it or put a tab indent (and do the same for all heredocs)